charts/falco/templates/pod-template.tpl

```diff
@@ -80,16 +70,6 @@ spec:
         - --cri
         - /run/crio/crio.sock
         {{- end }}
-        {{- if .kubernetes.enabled }}
-        - -K
-        - {{ .kubernetes.apiAuth }}
-        - -k
-        - {{ .kubernetes.apiUrl }}
-        {{- if .kubernetes.enableNodeFilter }}
-        - --k8s-node
-        - "$(FALCO_K8S_NODE_NAME)"
-        {{- end }}
-        {{- end }}
```

Comment:
I cannot comment on the line below https://github.com/falcosecurity/charts/pull/601/files#diff-52c2f29eec78e54b79385df867e0018793b671c77f7baea4563129e78447d9b1R84-R87 but there is a usage of `FALCO_BPF_PROBE` that probably we need to remove since it is now deprecated and we use the Falco config. Otherwise, we face the following logs:
```
Mon Jan 22 13:59:17 2024: Falco version: 0.37.0-rc1 (x86_64)
Mon Jan 22 13:59:17 2024: Falco initialized with configuration file: /etc/falco/falco.yaml
Mon Jan 22 13:59:17 2024: Since the new 'engine' config key is being used, deprecated CLI options [-e,-g,--gvisor-config,--nodriver,--modern-bpf] and 'FALCO_BPF_PROBE' environment variable will be ignored.
Mon Jan 22 13:59:17 2024: System info: Linux version 6.1.58+ (builder@69b9a3dc87a9) (Chromium OS 16.0_pre484197_p20230405-r10 clang version 16.0.0 (/var/tmp/portage/sys-devel/llvm-16.0_pre484197_p20230405-r10/work/llvm-16.0_pre484197_p20230405/clang 2916b99182752b1aece8cc4479d8d6a20b5e02da), LLD 16.0.0) #1 SMP PREEMPT_DYNAMIC Sat Dec 30 15:31:26 UTC 2023
Mon Jan 22 13:59:17 2024: Loading rules from file /etc/falco/falco_rules.yaml
Mon Jan 22 13:59:17 2024: The chosen syscall buffer dimension is: 8388608 bytes (8 MBs)
Mon Jan 22 13:59:17 2024: Starting health webserver with threadiness 2, listening on 0.0.0.0:8765
Mon Jan 22 13:59:17 2024: Loaded event sources: syscall
Mon Jan 22 13:59:17 2024: Enabled event sources: syscall
Mon Jan 22 13:59:17 2024: Opening 'syscall' source with BPF probe. BPF probe path: /root/.falco/falco-bpf.o
```